_posts/2022-11-19-sr2023-kickstart-and-rules.md

```diff
@@ -0,0 +1,25 @@
+---
+title: SR2023 Game Announced
+---
+
+Student Robotics 2023 was Kickstarted on Saturday 22nd September at the University of Southampton and on our [livestream][livestream]. It was great to see the excitement building at our first in-person Kickstart since SR2019.
+
+Our game this year, _Greed_, challenges teams to steal tokens from other scoring zones. With Bronze, Silver, and Gold tokens, each worth different amount of game points, teams must think carefully about the optimum strategy. To help teams locate tokens they have 2D barcode style markers attached that our computer vision library can detect. However each of the markers will identify themselves as exactly the same, so teams must use other sensors to detect which type of token is which. Bronze and Silver tokens are the same size but the Silver tokens weigh 300g more, and Gold Tokens are larger than Bronze and Silver. Teams are scored at the end of the match for collecting the most points. Full details, including the prizes available this year, are available in the [rulebook][rules].
+
+This year we have also introduced a new way for teams to earn league points, Challenges! There are three challenges which teams may optionally complete during the competition year in order. These challenges encourage teams to start work on their robots early and cover movement, sensors, and vision. The challenges may be approached in any order, and completing challenges before certain deadlines will earn the teams bonus league points. Full details of the challenges are available in the [rulebook][rules].
+
+This year's teams have already been strategising their approaches, and we can't wait to see their progress over the year!
+
+If you weren’t able to attend Kickstart this year, or would like a recap, you can:
+
+- watch our [livestream][livestream] where we explain the game and how this year's competition will work.
+- download [the presentation][presentation] in which we explain this year’s game and how the competition is running this year.
+- download [the microgames][microgames] which are small activities designed to let you become familiar with the Student Robotics simulator.
+
+If you're a competitor, be sure to check out our [Kickstarted, now what?][now-what] blog post for some next steps.
+
+[rules]: https://studentrobotics.org/docs/rules/
+[livestream]: https://www.youtube.com/watch?v=FVnNv5GSpyQ
+[presentation]: {{ site.baseurl }}/resources/sr2023/kickstart-sr2023.pdf
+[microgames]: {{ site.baseurl }}/resources/sr2023/microgames-sr2023.pdf
```

Comment:
I do wonder if the microgames should be migrated into the docs, especially as they're cross linked from the tutorials section. I realise that they've been in the website repo in the past, however I'm not sure that's the best place for them to end up.

Happy to take this discussion to Slack; this isn't a blocker either way as we can change this later easily enough.

Reply:
👍 have started a thread so we don't forget about this https://studentrobotics.slack.com/archives/CCD59522G/p1668893589131909